administration/src/bp-modules/applications/ApplicationCard.tsx

```diff
@@ -164,43 +192,44 @@ const ApplicationCard = ({
       }
     },
   })
+  const [approveStatus, approveStatusResult] = useApproveApplicationStatusMutation({
+    onError: error => {
+      const { title } = getMessageFromApolloError(error)
+      appToaster?.show({ intent: 'danger', message: title })
+    },
+    onCompleted: result => {
+      onChange({ ...application, status: result.status })
+    },
+  })
+
   const createCardQuery = useMemo(
     () =>
-      `${config.applicationFeature?.applicationJsonToCardQuery(jsonValueParsed)}&applicationIdToMarkAsProcessed=${id}`,
-    [config.applicationFeature, jsonValueParsed, id]
+      `./cards/add${config.applicationFeature?.applicationJsonToCardQuery(jsonValueParsed)}` +
```

Comment:
`applicationJsonToCardQuery` may return null. In which case the popup below (or maybe a simplified error à la "Karte kann nicht automatisch erstellt werden." should be shown. It may happen if the structure of the application form changes (older applications can not be migrated to a new application form structure).

Reply:
Alright thx for the hint. I would then tend to use a less cryptic message and better Show a toast if it can not be resolved

Reply:
What options do administrators have in such a case beside deleting the application?

Reply:
I'm not sure what you mean by "options", but they can still view the application, and if desired, manually create a card. (I don't know what is possible wrt. Freinet-related functionality)